tasks/codex/edm.py

```diff
@@ -228,3 +228,13 @@ def transfer_data(context, section=None):
             transfer_data_encounter()
         elif sect == 'sighting':
             transfer_data_sighting()
+
```

Comment:
This moved the call to User.edm_sync_all. 
What I meant was that the functionality in User.edm_sync_all needs to be moved into here. 
I'll sort that out too.

Reply:
ok.  btw `User.edm_sync_all()` _was_ there already.  i just moved _the task_ (that pointed there).  but end result the same!